eng/versioning/version_client.txt

```diff
@@ -55,22 +55,23 @@ com.azure:azure-spring-boot-test-core;1.0.0;1.0.0
 com.azure:azure-spring-boot-test-cosmosdb;1.0.0;1.0.0
 com.azure:azure-spring-boot-test-keyvault;1.0.0;1.0.0
 com.azure:azure-spring-data-gremlin;2.3.1-beta.1;2.3.1-beta.1
-com.azure.resourcemanager:azure-resourcemanager;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-appplatform;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-appservice;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-authorization;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-compute;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-containerregistry;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-containerservice;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-cosmos;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-dns;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-keyvault;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-monitor;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-msi;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-network;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-resources;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-sql;2.0.0-beta.4;2.0.0-beta.4
-com.azure.resourcemanager:azure-resourcemanager-storage;2.0.0-beta.4;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-appplatform;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-appservice;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-authorization;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-compute;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-containerinstance;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-containerregistry;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-containerservice;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-cosmos;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-dns;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-keyvault;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-monitor;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-msi;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-network;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-resources;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-sql;2.0.0-beta.3;2.0.0-beta.4
+com.azure.resourcemanager:azure-resourcemanager-storage;2.0.0-beta.3;2.0.0-beta.4
```

Comment:
May need to double check with Jim on why previously both is beta.4.

Reply:
I think he just do a update, not check too much about the version.

Reply:
beta.3 wasn't released and there's nothing in the repository, as of yet, that uses the dependency version of any of these libraries. I initially set them both current and dependency to beta.4 since that was the next version after what you'd planned to release this week. Updating the dependency here, after the release of beta.3, is the correct course of action. Now, if anything starts to use the dependency version it'll use the released version.